src/App.js

```diff
@@ -22,12 +22,21 @@ function App() {
     setTodoList([...todoList, newTodo]);
   }
 
+  const removeTodo = (id) => {
+    const item = todoList.find(element => element.id === id);
+    console.log(item);
+    const itemIndex = todoList.indexOf(item);
+    todoList.splice(itemIndex, 1);
+    const updatedTodoList = [].concat(todoList);
+    setTodoList(updatedTodoList);
+  }
```

Comment:
This will work fine, but it's quite a lot of lines and does quite a lot of work for something quite simple.

A way to do the same thing in a way that is much easier to read is:
```
const newTodoList = todoList.filter(item => item.id !== id);

setTodoList(newTodoList);
```